src/rules/converters/class-name.ts

```diff
@@ -4,7 +4,7 @@ export const convertClassName: RuleConverter = () => {
     return {
         rules: [
             {
-                ruleName: "@typescript-eslint/class-name-casing",
+                ruleName: "@typescript-eslint/naming-convention",
```

Comment:
[TSLint's `class-name`](https://palantir.github.io/tslint/rules/class-name) only allows PascalCase class names. Reading through the `naming-convention` source, I'm not seeing anything that defaults class names to any particular case: PascalCase or otherwise. That means this'll need some rule options to replicate the old behavior, right?

Reply:
Based on the docs, the [`default` selector](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#group-selectors) matches everything but it is marked as camelCase, so you got a point here.

Reply:
@JoshuaKGoldberg Actually, the PascalCase for Classes and Interfaces is set as default. Checkout https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/rules/naming-convention.ts#L331 and the docs at [options](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#options) (scroll down a bit).